### ISSUE TITLE
Refactor the "Your gist is ready" modal dialog

### ISSUE THREAD

**Issue by lolmaus** [jedfoster/SassMeister#58]
Currently the "Your gist is ready" modal dialog contains two links:
- a link to the corresponding gist;
- a link to the current SassMeister project.

Both links are bad, and the dialog should feel bad.

**The link to gist should be moved to the "cloud" menu** where it belongs. That's where any person would look for it.

Also, there's a big problem with the link to gist being unavailable. When a user opens another user's SassMeister project and wants visit its gist, there is no way he can do that. He can only fork the project, then he will be able to click the link in the "Your gist is ready" modal dialog. But it will be a different gist! What if he wants to comment on the original gist or learn about its author?

**The link to the current SassMeister project** should not exist in the first place. Its problem is that it forces the page to reload (for me it's 13 requests, ~250KB and five seconds; why isn't app.js being cached?). But why reload in the first place? It brings me to absolutely the same page where i was before i clicked the link! Also, SassMeister updates its URL, so i can just copy it from the browser's URL bar.

TL/DR:
1. Add a "Go to gist" link to the "cloud" menu.
2. Replace the "Your gist is ready" modal dialog with a non-blocking and self-dismissing notification. It should not contain a link that causes SassMeister to reload.

PS This is just a suggestion for you to consider, i wasn't intending to be bossy.
